src/test/java/StepDefinitions/LoginSteps.java

```diff
@@ -0,0 +1,83 @@
+package StepDefinitions;
+
+import PageFactory.HomePage;
+import PageFactory.LoginPage;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class LoginSteps {
+    WebDriver driver;
+    WebDriverWait wait;
+    @Before
+    public void openBrowser() {
+        WebDriverManager.chromedriver().setup();
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--disable-notifications");
+        options.addArguments("--remote-allow-origins=*");
+        driver = new ChromeDriver(options);
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+    @After
+    public void closeBrowser(){
+        driver.quit();
+    }
+    @Given("I open Login page")
+    public void openLoginPage() {
+        driver.get("https://qa.koel.app/");
+    }
+
+    // Without PAGE OBJECTS
+
+//    @When("I enter email {string}")
+//    public void i_enter_email(String email) {
+//        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[type='email']"))).sendKeys(email);
+//    }
+//    @And("I enter password {string}")
+//    public void i_enter_password(String password) {
+//        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[type='password']"))).sendKeys(password);
+//    }
+//    @And("I submit")
+//    public void clickSubmit() {
+//        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[type='submit']"))).click();
+//    }
+//    @Then("I am logged in")
+//    public void userIsLoggedIn() {
+//        Assert.assertTrue(wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("img.avatar"))).isDisplayed());
+//    }
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.